src/core/DrumSynth.cpp

```diff
@@ -398,7 +390,7 @@ int DrumSynth::GetDSFileSamples(QString dsfile, int16_t*& wave, int channels, sa
 	GetEnv(7, sec, "FilterEnv", dsfile);
 
 	// read noise parameters
-	strcpy(sec, "Noise");
+	std::memcpy(sec, "Noise", sizeof("Noise"));
```

Comment:
The same can be said for this one. You can also move the magic constants (`"Noise` in this case) into a `constexpr` variable if possible to avoid mistakes.

```suggestion
	strncpy(sec, "Noise", strlen("Noise") + 1);
```